frontend/src/app/api/BaseApi.ts

```diff
@@ -33,9 +39,17 @@ export default abstract class BaseApi {
   // Can include feature flags in child classes
   get headers(): HeadersDict {
     const headers: HeadersDict = {};
-    if (process.env.NEXT_PUBLIC_LOCAL_AUTH_TOKEN) {
+
+    if (
+      process.env.NODE_ENV === "development" &&
+      process.env.NEXT_PUBLIC_LOCAL_AUTH_TOKEN
+    ) {
       headers["X-AUTH"] = process.env.NEXT_PUBLIC_LOCAL_AUTH_TOKEN;
     }
+
+    if (process.env.NODE_ENV === "production" && process.env.API_AUTH_TOKEN) {
+      headers["X-AUTH"] = process.env.API_AUTH_TOKEN;
+    }
```

Comment:
I don't believe this will work (but it might) - I might need to work with @coilysiren  on the permissions for this

Reply:
Why would there be a check for environment here? Every environment, including local should also fetch the same way, right?

I see for local you set it up different, why not just set everything to the same environment variable? Otherwise you need to configure every environment differently

Reply:
good catch! Updated the `.env.development` to include the public auth token as `API_AUTH_TOKEN` so we don't have to check env here